tools/install.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import os
+import json
+import sys
+import tempfile
+import shutil
+from urllib2 import urlopen
```

Comment:
This file can be python3 compatible:
```
from __future__ import print_function
try:
    from urllib.request import urlopen
except ImportError:
    from urllib2 import urlopen
```
and updating the print statements should suffice.

Reply:
As soon as I get it working on all platforms I will make it p3 compatible @hayd